spec/time_stack_item_spec.cr

```diff
@@ -1,30 +1,29 @@
 require "./spec_helper"
 
 describe Timecop do
-
   it "new_with_time" do
-    t = Time.now
+    t = Time.local
     y, m, d, h, min, s = t.year, t.month, t.day, t.hour, t.minute, t.second
     stack_item = Timecop::TimeStackItem.new(:freeze, t)
 
-    true.should eq(y == stack_item.year)
-    true.should eq(m == stack_item.month)
-    true.should eq(d == stack_item.day)
-    true.should eq(h == stack_item.hour)
-    true.should eq(min == stack_item.minute)
-    true.should eq(s == stack_item.second)
+    y.should eq(stack_item.year)
+    m.should eq(stack_item.month)
```

Comment:
Do we even need these intermediate variables for hour, min, etc? Why not just `t.hour.should eq(stack item.hour)`

Reply:
Good point, will do.